magicctapipe/scripts/lst1_magic/lst1_magic_cloud_correction.py

```diff
@@ -0,0 +1,425 @@
+#!/usr/bin/env python
```

Comment:
there is still a mess up with some of the correction in the previous file and added a second file. this needs to be cleared up before the code can be merged.